### PR TITLE
Link skills, plug effects, tower block names, some music, other fixes

### PR DESCRIPTION
**Pull request by DabirSA**

### CODE REVIEW

File: json/TowerQuest_BlockSettingText.txt [hunk 1]
```diff
@@ -5,55 +5,55 @@
 	},
 	{
 		"jp_text": "<color=red>▼</color>５秒毎にＨＰが <%parameter1%>％減少",
-		"tr_text": ""
+		"tr_text": "<color=red>▼</color>Lose <%parameter1%>％ HP every 5 seconds"
```
Comment (alama) [inline json/TowerQuest_BlockSettingText.txt]: `％ HP every 5 seconds` or `% HP every 5 seconds`?

File: json/TowerQuest_BlockSettingText.txt [hunk 2]
```diff
@@ -5,55 +5,55 @@
...
+		"tr_text": "<color=red>▼</color>Support Chips don't activate"
 	},
 	{
 		"jp_text": "[<%parameter1%>]（30秒毎発動）",
 		"tr_text": ""
 	},
 	{
 		"jp_text": "<color=#00ffff>▲</color>状態異常の敵に与えるダメージが <%parameter1%>％に",
-		"tr_text": ""
+		"tr_text": "<color=#00ffff>▲</color><%parameter1%>％ boosted damage against statused enemies"
```
Comment (alama) [inline json/TowerQuest_BlockSettingText.txt]: `％ boosted damage against statused enemies` or `% boosted damage against statused enemies`?

File: json/TowerQuest_BlockSettingText.txt (hunk 1 above)
Comment (DabirSA) [reply json/TowerQuest_BlockSettingText.txt]: Fixed

File: json/TowerQuest_BlockSettingText.txt (hunk 2 above)
Comment (DabirSA) [reply json/TowerQuest_BlockSettingText.txt]: Fixed

File: json/Item_Stack_Music.txt [hunk 3]
```diff
@@ -249,28 +249,28 @@
...
-		"tr_explain": ""
+		"tr_explain": "A music disc for your jukebox.\nPSO2 #56 (Melody can be changed)\n\"Mining Base Defense (Danger)\""
 	},
 	{
 		"assign": "319037",
 		"jp_text": "採掘基地防衛戦3",
 		"tr_text": "Mining Base Defense 3",
 		"jp_explain": "ジュークボックス用音楽再生ディスク。\nＰＳＯ２ ＃５７（曲調変更可）\n『採掘基地防衛戦（通常＋AIS）』",
-		"tr_explain": ""
+		"tr_explain": "A music disc for your jukebox.\nPSO2 #57 (Melody can be changed)\n\"Mining Base Defense (Normal + AIS)\"""
```
Comment (alama) [inline json/Item_Stack_Music.txt]: One too many `"`, again? Double check all your new strings?

**Pull request merged**